app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
```

Comment:
If you used `for index, arg in enumerate(...)` as suggested above, then after refactoring it could have been simpler:

```suggestion
        if arg == ["-f"]:
```